test/dom/attr.test.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const { describe, test } = require('@jest/globals');
+const { Attr } = require('../../lib/dom');
+
+describe('Attr.prototype', () => {
+	describe('constructor', () => {
+		test('should throw Illegal constructor TypeError when trying to access constructor directly', () => {
+			expect(new Attr('attr')).toThrow(TypeError);
```

Comment:
In order to use `.toThrow` you have to wrap the throwing code into a function, so it can be executed outside in a try catch as part of the assertion:
```suggestion
			expect(() => new Attr('attr')).toThrow(TypeError);
```